### ISSUE TITLE
wrong dimensions ?

### ISSUE THREAD

**Issue by pjbishop93** [paulh-rnd/TabbedBoxMaker#52]
I have inkscape 1.2. I create a 1" x 1" x 1" box with .5" fixed width tabs, .125" thick material, 0.06" kerf. It creates a 2.813" box. The tabs are the correct size. No matter what size I make the box, it seems to be off by 1.8". Maybe I'm not understanding about how to set the parameters.

<img width="574" alt="Screen Shot 2022-10-29 at 9 26 11 AM" src="https://user-images.githubusercontent.com/87703646/198834053-df2bb625-4777-4669-b57e-066894e1e83f.png">

<img width="550" alt="Screen Shot 2022-10-29 at 9 28 41 AM" src="https://user-images.githubusercontent.com/87703646/198834170-802b13d7-3c9c-49ed-b437-fb236e53685a.png">

**Comment by RadioActiveLamb** [comment]
I have the same problem. I created a 6x6x1.5" box, but the resulting box is about 9". Inkscape 1.2.
If I convert all the dimensions to mm, then the box is the correct size.

**Comment by WagonWillie** [comment]
I also have the same problem: a 15"x15"x3" box turns out to be an 18.689"
x18.689" x 6.689" box with tabs that will not line up.  However, a 360mm x
360mm x 75mm box with 10mm tabs turns out very close to those dimensions
and with tabs that work.

**Comment by pjbishop93** [comment]
I discovered after I posted the question that it wasn't converting to inches. Hopefully someone can fix it. In the meantime, metric it is.

**Comment by WagonWillie** [comment]
Yes, I have no problem with metric, except this order was expressed in
inches, so that's what I tried.
